data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/CompositeException.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * @since 1.3
+ * <p>
+ * Exception thrown by {@link ParseTreeParser} if ANTLR parse emits error events.
+ */
+public class CompositeException extends RuntimeException {
+    private static final String SEPARATOR = System.getProperty("line.separator");
+
+    private final Set<Throwable> exceptions;
+
+    public CompositeException(final List<Throwable> exceptions) {
+        if (exceptions.isEmpty()) {
+            throw new IllegalArgumentException("errors is empty");
```

Comment:
Should this say, "exceptions is empty"?

Reply:
Thanks for the suggestions, I'll update the message.